src/mappings/handlers/generic.ts

```diff
@@ -1,16 +1,42 @@
 import { SubstrateExtrinsic } from "@subql/types";
 import { handleMultisigCall } from "./multisigCallHandler";
 import { handleRemark } from "./multisigRemarkHandler";
-import { handleProxyCall } from "./proxyCallHandler";
+import { handleRemoveProxiesCall } from "./proxyCallHandler";
+import { CreateCallVisitorBuilder, CreateCallWalk } from "subquery-call-visitor";
 
-export function handleProxyProxy(extrinsic: SubstrateExtrinsic) {
-  return Promise.all([handleRemark(extrinsic), handleProxyCall(extrinsic)]);
-}
+const callWalk = CreateCallWalk();
 
-export function handleMultisig(extrinsic: SubstrateExtrinsic) {
-  return Promise.all([handleMultisigCall(extrinsic), handleRemark(extrinsic), handleProxyCall(extrinsic)]);
-}
+const visitor = CreateCallVisitorBuilder()
+  .on("utility", ["batch", "batchAll", "forceBatch"], (extrinsic, context) => {
+    const calls = extrinsic.call.args.at(0);
+    if (Array.isArray(calls) && calls.length > 100) {
+      // we're skipping large batches, something terrible happens inside anyway
+      context.stop();
+    }
+  })
```

Comment:
also, we've checked batches on events lenght, would be better to add it here:
```suggestion
    const maxLength = 10_000;
    const calls = extrinsic.call.args[0];
    // we're skipping large batches, something terrible happens inside anyway
    if (Array.isArray(calls) && calls.length > maxLength || extrinsic.events?.length > maxLength) {
      context.stop();
    }
  })
```

Reply:
and lets increase a maxLenght, looks like it may pretty legit for batch with 1000 events